core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -3050,9 +3059,9 @@ void ExHbaseAccessTcb::buildLoggingPath(
      struct tm * curgmtime = gmtime(&t);
      strftime(logId_tmp, sizeof(logId_tmp)-1, "%Y%m%d_%H%M%S", curgmtime);
      logId = logId_tmp;
-  }
-  sprintf(loggingFileName, "%s/ERR_%s/%s_%s_%d",
-                  loggingLocation, logId, tableName, loggingFileNamePrefix, instId);
+  } 
+  sprintf(currCmdLoggingLocation, "%s/ERR_%s_%s",
```

Comment:
The loggingLocation_ is allocated in line 1889 in setLoggingLocation_ with large enough buffer.